x/launch/simulation/simulation.go

```diff
@@ -228,36 +240,37 @@ func SimulateMsgRequestRemoveAccount(ak types.AccountKeeper, bk types.BankKeeper
 			accChainID uint64
 		)
 		found := false
-		for acc, chainID := range accsChain {
-			if IsLaunchTriggeredChain(ctx, k, chainID) {
+		for _, accChain := range accChainList {
```

Comment:
It will find the first account in the list that fulfills all conditions

I suggest we add a shuffling in the list `r.Suffle` to add more entropy otherwise there is a high change only genesis account and not vesting account will be fetched